xonsh/main.py

```diff
@@ -253,6 +254,33 @@ def premain(argv=None):
 
 
 def main(argv=None):
+    try:
+        return main_xonsh(argv)
+    except Exception:
+        traceback.print_exc()
```

Comment:
How should I know xt is still available? after all it's doing fail over. So I guess use `traceback` module is safer?